iphone/Classes/TiUITableView.m

```diff
@@ -312,6 +312,11 @@ - (id)init
 
 - (void)dealloc
 {
+  if ([searchController isActive]) {
+    searchController.view.hidden = YES;
+    [searchController setActive:NO];
```

Comment:
Why not remove it's delegates and nil it as well (like the others)?

Reply:
That is already there. See few lines below in this function :) . You will find -

    searchController.searchResultsUpdater = nil;
   searchController.delegate = nil;
   RELEASE_TO_NIL(searchController);

Reply:
I see this for the list-view, but not the table-view. Am I missing a snippet?

Reply:
I think so. You can see at line 338.

Reply:
I mean list-view, I cannot see nil'ing `searchResultsUpdater ` in `TiUIListView.m`.

Reply:
Ok. You are right. Done.